pom.xml

```diff
@@ -193,9 +195,11 @@
             <module>plugins/jgroups32</module>
             <module>plugins/jgroups33</module>
             <module>plugins/jgroups34</module>
+            <module>plugins/jgroups35</module>
+            <module>plugins/jgroups36</module>
          </modules>
          <properties>
-            <plugins.jgroups>jgroups30,jgroups32,jgroups33,jgroups34</plugins.jgroups>
+            <plugins.jgroups>jgroups30,jgroups32,jgroups33,jgroups34,jgroups35,jgroups36</plugins.jgroups>
```

Comment:
JGroups 3.5 has its own profile few lines below. If you're including 3.5 to the default build, please remove that profile and set JGroups dependency in jgroups35 plugin to latest release from 3.5 branch.